subsys/bluetooth/host/buf.c

```diff
@@ -85,7 +85,6 @@ struct net_buf *bt_buf_get_cmd_complete(k_timeout_t timeout)
 
 	key = irq_lock();
 	buf = bt_dev.sent_cmd;
```

Comment:
Also should return net_buf_ref(buf)
If send_cmd not null, right? @jhedberg

Reply:
This line should be:
`buf = net_buf_ref(bt_dev.sent_cmd)`

Reply:
`bt_dev.sent_cmd` maybe null ?

Reply:
Done.